src/app/views/sidebar/resource-explorer/ResourceLink.tsx

```diff
@@ -27,6 +28,16 @@ const ResourceLink = (props: IResourceLinkProps) => {
   const paths = collections?.find(k => k.isDefault)?.paths || [];
   const resourceLink = { ...link };
 
+  const [alwaysShowButtons, setAlwaysShowButtons] = useState(false);
+
+  useEffect(() => {
+    variantService.getFeatureVariables('default', 'alwaysShowButtons').then((value) => {
+      if(value !== undefined) {
```

Comment:
This condition can be simplified to `if (value)`, which will check for any falsy value, not just undefined. This will make the code more concise and readable.

Reply:
This value is allowed to be false though.... the experiment randomizes the response, some people will get true and some will get false...
In cases where initialization was slow or there is a problem somehwere, it would return undefined and use the default state